app-text/calibre/calibre-5.32.0.recipe

```diff
@@ -175,8 +178,10 @@ BUILD_REQUIRES="
 	devel:libpoppler${secondaryArchSuffix}_qt5
 	devel:libpoppler$secondaryArchSuffix
 	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5WebEngine$secondaryArchSuffix
 	devel:libQt5WebKit$secondaryArchSuffix
```

Comment:
devel:libQt5WebKit can probably be removed?

Reply:
Yep